consensus/wal.go

```diff
@@ -20,6 +20,7 @@ import (
 
 const (
 	// amino overhead + time.Time + max consensus msg size
+	// TODO: Can we clarify better where 24 comes from precisely?
```

Comment:
so cdc.MustMarshalBinaryBare(empty consensus part msg) = 14 bytes. + 10 bytes just in case amino will require more space in the future. who knows

Reply:
Lol ok that should be documented in the code ideally :)